DataLayer/gradle/libs.versions.toml

```diff
@@ -1,27 +1,27 @@
 [versions]
-android-gradle-plugin = "8.10.0"
-androidx-activity = "1.10.1"
-androidx-compose-bom = "2025.05.01"
-androidx-lifecycle = "2.9.0"
-androidx-wear-compose = "1.5.0-beta01"
+android-gradle-plugin = "8.12.0"
+androidx-activity = "1.12.0-alpha06"
```

Comment:
as above, does it need alpha?

Reply:
same as above, this fixes failure with Roborazzi tests as core-ktx is a transitive dependency